icepyx/core/granules.py

```diff
@@ -182,29 +192,26 @@ def place_order(self, CMRparams, reqparams, subsetparams, verbose,
         base_url = 'https://n5eil02u.ecs.nsidc.org/egi/request'
         #DevGoal: get the base_url from the granules?
 
-
-        self.get_avail(CMRparams, reqparams) #this way the reqparams['page_num'] is updated
+        self.get_avail(CMRparams, reqparams)
 
         if subset is False:
-            request_params = apifmt.combine_params(CMRparams, reqparams, {'agent':'NO'})
+            request_params = apifmt.combine_params(CMRparams, reqparams, {'agent': 'NO'})
         else:
             request_params = apifmt.combine_params(CMRparams, reqparams, subsetparams)        
         
         order_fn = '.order_restart'
         
-        print('Total number of data order requests is ',request_params['page_num'], ' for ',len(self.avail), ' granules.')
-        #DevNote/05/27/20/: Their page_num values are the same, but use the combined version anyway.
-        #I'm switching back to reqparams, because that value is not changed by the for loop. I shouldn't cause an issue either way, but I've had issues with mutable types in for loops elsewhere.
-        for i in range(reqparams['page_num']):
-#         for i in range(request_params['page_num']):
-            page_val = i + 1
-            
-            print('Data request ', page_val, ' of ', reqparams['page_num'],' is submitting to NSIDC')
-            request_params.update( {'page_num': page_val} )
-
-            #DevNote: earlier versions of the code used a file upload+post rather than putting the geometries
-            #into the parameter dictionaries. However, this wasn't working with shapefiles, but this more general
-            #solution does, so the geospatial parameters are included in the parameter dictionaries.
+        total_pages = int(np.ceil(len(self.avail) / reqparams['page_size']))
+        print('Total number of data order requests is ',total_pages, ' for ',len(self.avail), ' granules.')
+        for page_num in range(1, total_pages+1):
+  
+            print('Data request ',page_num,' of ',total_pages,' is submitting to NSIDC',)
+            request_params = apifmt.combine_params(CMRparams, reqparams, subsetparams)
```

Comment:
I think this line is redundant with line 200 and will result in unintentional subsetting if subset=False. Suggest removing.

Reply:
Remove line (and help @JessicaS11 learn how to use the GitHub PR review tools!)

Reply:
We'll have to learn together -- I predominantly use bitbucket at work so github is much less familiar to me!